preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -196,23 +200,6 @@ def process_date(
         warnings = []
         errors = []
 
-        # TODO: required check is also in process_single - check if can be removed here
```

Comment:
have you checked that we can remove the required logic? I assume so but a quick confirmatory comment would be great :)

Reply:
Thanks! It's not quite clear yet how this is related to tests though, we're removing some functionality here from the actual code - do you mean that you verified through a test that this code here was no longe required?

Reply:
yes exactly - the code here is not required as this check is performed at another location. Additionally I added a test to confirm this code is indeed not required